broker/broker_test.go

```diff
@@ -43,14 +40,12 @@ var _ = Describe("Broker", func() {
 				AccessKeyID:     "access-key!",
 				SecretAccessKey: "secret-key!",
 			})
-			Expect(uri).To(Equal("s3://access-key%21:secret-key%21@s3.amazonaws.com/bucket"))
+			Expect(uri).To(Equal("s3://access-key%21:secret-key%21@s3-fips.amazonaws.com/bucket"))
```

Comment:
After the refactor, these tests are failing and need to be updated like so:

```
			uri := broker.GetBucketURI(Credentials{
				Bucket:          "bucket",
				AccessKeyID:     "access-key!",
				SecretAccessKey: "secret-key!",
				FIPSEndpoint:    "s3-fips.us-east-1.amazonaws.com",
			})
			Expect(uri).To(Equal("s3://access-key%21:secret-key%21@s3-fips.us-east-1.amazonaws.com/bucket"))
```

Reply:
darn it. sorry about that. I ran the main package tests rather than entire broker. My go is pretty rusty.